src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.LinkedHashMap;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
```

Comment:
Using BufferedWriter directly with FileWriter without specifying a charset can lead to platform-dependent behavior. It is recommended to use Files.newBufferedWriter with a charset, for example, StandardCharsets.UTF_8, to ensure consistent behavior across platforms.